front/src/modules/pathfinding/hooks/usePathfinding.ts

```diff
@@ -324,6 +324,7 @@ export const usePathfinding = (
                     name: correspondingOp.name,
                     uic: correspondingOp.uic,
                     ch: correspondingOp.ch,
+                    secondary_code: correspondingOp.ch,
```

Comment:
Hm. I'm worried these two fields go out-of-sync.

Reply:
Hm, well, I think if we remove this `getStepLocation()` will break. In other words, if half of the codebase uses `ch` and the other half uses `secondary_code`, we get fireworks.

Reply:
https://github.com/OpenRailAssociation/osrd/pull/8633 has been merged now, so this concern is alleviated :)

(Sorry about the conflicts…)

Reply:
Is there a reason why we can't rename ch into secondary code for every single type? Are there cases where there's a meaningful difference?

Reply:
No real reason apart from this being a sizable amount of work (some of the types are generated from OpenAPI). But yeah, we should aim to gradually rename everything.

Reply:
No pb for the conflicts ! :+1: